core/src/main/kotlin/io/github/serpro69/kfaker/Faker.kt

```diff
@@ -201,7 +201,7 @@ class Faker @JvmOverloads constructor(internal val fakerConfig: FakerConfig = Fa
         beer = Beer(fakerService)
         bigBangTheory = BigBangTheory(fakerService)
         blood = Blood(fakerService)
-        bojackHoreseman = BojackHoreseman(fakerService)
+        bojackHorseman = BojackHorseman(fakerService)
```

Comment:
Can you add a new property that will live along the old one instead of replacing it? And add `@Deprecated` to the old one?
It's fine to "replace" things with correct names in documentation, and in the cli-bot as well, but I would prefer not to introduce breaking changes to the "core" without deprecating the old stuff.